src/blog/2022/11/flowforge-1-1-released.md

```diff
@@ -0,0 +1,92 @@
+---
+title: FlowForge 1.1 released
+subtitle: In FlowForge 1.1, we are continuing to add the features we believe makes FlowForge the best solution for running Node-RED in production in a secure, scalable, and team-based environment.
+description: In FlowForge 1.1, we are continuing to add the features we believe makes FlowForge the best solution for running Node-RED in production in a secure, scalable, and team-based environment.
+date: 2022-11-24 18:00:00.0
+authors: ["rob-marcer"]
+video: TODO
+---
+
+Persist files on your FlowForge projects, import Node-RED snapshots into FlowForge directly from the Node-RED editor and use our new interface for managing Project Deployments
```

Comment:
The messaging around importing snapshots I think doesn't cover what users will use this for.

We now allow everyone to develop locally on the bench and push updates to all the project deployments. The mechanics under the hood are through snapshots, sure. But that's not the value we offer.

Reply:
Also, a sentence ends with a full stop

Reply:
Missing oxford comma too